File: app/src/main/java/com/script972/clutchclient/ui/model/CardItem.java
package com.script972.clutchclient.ui.model;

import lombok.Data;

@Data
public class CardItem {

    private long id;

    private String title;

    private String cardNumber;

    private String comment;

    private String photoFront;

    private String photoBack;

    private boolean updateMode;
}

File: app/src/main/java/com/script972/clutchclient/ui/model/CardItem.kt
package com.script972.clutchclient.ui.model

data class CardItem(

        var id: Int = 0,

        var title: String? = null,

        var cardNumber: String? = null,

        var comment: String? = null,

        var photoFront: String? = null,

        var photoBack: String? = null,

        var updateMode: Boolean = false,

        var listAccessUser: MutableList<AccessPersonItem>? = ArrayList()

)
